src/components/modal/index.js

```diff
@@ -9,6 +9,7 @@ export const Modal = ({ title, open, onClose, children }) => {
     const styles = StyleSheet.create({
         root: {
             backgroundColor: theme.background,
+            padding: 5,
```

Comment:
Why? 

P.S.: I follow a "multiple of 4" convention for spacing.

Reply:
The modal should have a default padding, the text was "touching" the borders :nauseated_face:.